File: remote/src/main/java/com/kelsos/mbrc/presenters/MiniControlPresenter.java
package com.kelsos.mbrc.presenters;

import com.kelsos.mbrc.ui.views.MiniControlView;

public interface MiniControlPresenter {
  void onNextPressed();
  void onPreviousPressed();
  void onPlayPause();
  void bind(MiniControlView view);
  void onResume();
  void onPause();
  void load();
}

File: remote/src/main/kotlin/com/kelsos/mbrc/presenters/MiniControlPresenter.kt
package com.kelsos.mbrc.presenters

import com.kelsos.mbrc.ui.views.MiniControlView

interface MiniControlPresenter {
  fun onNextPressed()
  fun onPreviousPressed()
  fun onPlayPause()
  fun bind(view: MiniControlView)
  fun onResume()
  fun onPause()
  fun load()
}
